test/runtests.jl

```diff
@@ -23,22 +16,24 @@ using Random
 using StaticArrays
 using Statistics
 using Test
+using JuliaInterpreter
 
-@nospecialize
-
-if isdefined(Base, :Experimental) && isdefined(Base.Experimental, Symbol("@compiler_options"))
-    @eval Base.Experimental.@compiler_options compile=min optimize=0 infer=no
-end
+union!(JuliaInterpreter.compiled_modules, Any[Base, Base.Broadcast, Compat, LinearAlgebra, Random, StaticArrays, Statistics])
 
 Random.seed!(1) # Set seed that all testsets should reset to.
 
 function include_test(path::String)
     println("Testing $path:")  # print so TravisCI doesn't timeout due to no output
-    @time include(path)  # show basic timing, (this will print a newline at end)
+    @time Base.include(@__MODULE__(), path) do ex
+        Meta.isexpr(ex, :macrocall) && ex.args[1] == Symbol("@testset") || return ex
+        return :(@interpret (() -> $ex)()) # interpret testsets using JuliaInterpreter
```

Comment:
BlueStyle likes 2 spaces for comments on same line
```suggestion
        return :(@interpret (() -> $ex)())  # interpret testsets using JuliaInterpreter
```